_RELEASE/Packs/cube/Scripts/utils.lua

```diff
@@ -1,19 +1,12 @@
--- initialize random seed
-math.randomseed(os.time())
-math.random()
-math.random()
-math.random()
-
 -- shuffle: shuffles an array
 function shuffle(t)
-	math.randomseed(os.time())
 	local iterations = #t
 	local j
 	for i = iterations, 2, -1 do
 			j = math.random(i)
 			t[i], t[j] = t[j], t[i]
 	end
-	
+
 	return t
 end
```

Comment:
Here i personally would go with:
```lua
function shuffle(t)
	for i = 1, #t do
		local j
		repeat j = math.random(#t) until i ~= j
		t[i], t[j] = t[j], t[i]
	end
end
```
So it prevents from swapping the same element from both i and j variables.
But it's not nessesary.